### DISCUSSION TITLE
主線圖功能出錯，只會一直點選擇隊伍，不會進入戰鬥

### DISCUSSION THREAD

**Discussion by hsd325** [LmeSzinc/AzurLaneAutoScript#3814] [General]
問題如標題那樣
遊戲裡設定都有更改
主線圖設定是調成這樣
![螢幕擷取畫面 (194)](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/100175482/a4d85c21-a6bb-4ba9-82b3-b3b388b232ff)

錯誤要給的log/error裡面圖片和文本
![2024-05-25_18-33-08-992653](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/100175482/dde2c22f-af0d-4008-84e6-aa06f8078e55)
[log.txt](https://github.com/LmeSzinc/AzurLaneAutoScript/files/15442785/log.txt)

更新
喔 了解了 是因為其他艦隊還沒開起的原因 要解決就必須打完第7章
我查資料，這個bug好像2024/1就有了
這個bug之後會修掉嗎?

**Comment by jeff875612** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/3429
